README.md

```diff
@@ -1 +1,34 @@
-# AI_Conference-CapstoneProject
\ No newline at end of file
+# AI_Conference-CapstoneProject
+  This Web page is a the first Capstone project for Microverse first module. Itis about the importance of Artificial Intelligence in our life. It designed according to Behance Cindy Shine Design.
+
+# Build With 👩‍🔧
+.HTML
+
+.CSS
+
+.JavaScript
+
+.Sublime
+
+# Getting Started 🎬
+In this project I create a Headline page, Cards, About me page. It has a menu bar which is expandable, a popup menu for both mobile and desktop virsion and button interaction.
+
+# Author
+Sahar Saba Amiri
+
+. Gmail: saharsaba.amiri123@gmail.com
+
+. LinkedIn: https://www.linkedin.com/sahar-saba-amiri
+
+. GitHub : https://github.com/Sahar-SE
+
+# Contributing
+Contributions, issues, and feature requests are welcome!
+
+# links
+The link of my Portfolio GitHub io is: 
```

Comment:
- [ ] Please record a presentation video using loom, as the google drive recording could not be played.

Reply:
- [ ] [OPTIONAL] Although not pointed out by the other reviewer, please make sure that the live link is working properly, at the moment it is only pointing towards the README file